Cargo.toml

```diff
@@ -25,10 +25,11 @@ default-logger = ["slog-stdlog", "slog-envlogger", "slog-term"]
 
 # Make sure to synchronize updates with Harness.
 [dependencies]
+bytes = "1.0"
 fxhash = "0.2.1"
 fail = { version = "0.3", optional = true }
 getset = "0.0.9"
-protobuf = "2"
+protobuf = "2.23"
```

Comment:
We are using a fork of protobuf, which may not be at the same version.

Reply:
protobuf 2.8 uses a stale version for `bytes`. I think we can upgrade our protobuf.